cli/src/main/java/hudson/cli/CLI.java

```diff
@@ -245,9 +245,9 @@ public static int _main(String[] _args) throws Exception {
 
         if (auth == null && bearer == null) {
             // -auth option not set
-            if (StringUtils.isNotBlank(userIdEnv) && StringUtils.isNotBlank(tokenEnv)) {
+            if ((userIdEnv != null && !userIdEnv.isBlank()) && (tokenEnv != null && !tokenEnv.isBlank())) {
```

Comment:
In the following line we can remove the usage of StringUtils.defaultString. We already know that both strings are not null and not empty
`auth = userIdEnv.concat(":").concat(tokenEnv)`

Reply:
> In the following line we can remove the usage of `StringUtils.defaultString`.

Yes, we can. I see no reason why we should do so in this PR.